RFC/src/RFC-0110_BaseNodes.md

```diff
@@ -22,132 +22,136 @@ following conditions are met:
+SPECIAL, EXEMPLARY OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+WHETHER IN CONTRACT, STRICT LIABILITY OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 ## Language
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", 
-"NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in 
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", 
+"NOT RECOMMENDED", "MAY" and "OPTIONAL" in this document are to be interpreted as described in 
 [BCP 14](https://tools.ietf.org/html/bcp14) (covering RFC2119 and RFC8174) when, and only when, they appear in all capitals, as 
 shown here.
 
 ## Disclaimer
 
-The purpose of this document and its content is for information purposes only and may be subject to change or update
+This document and its content are intended for information purposes only and may be subject to change or update
 without notice.
 
 This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
-community. The release of this document is intended solely for review and discussion by the community regarding the
+community. The release of this document is intended solely for review and discussion by the community of the
 technological merits of the potential system outlined herein.
 
 ## Goals
 
-This document describes the roles that [base node]s play in the Tari network and their general approach for doing so.
+The aim of this Request for Comment (RFC) is to describe the roles that [base node]s play in the Tari network as well as 
+their general approach for doing so.
 
-## Related RFCs
+## Related Requests for Comment
 
-* [RFC-0100: Base layer](RFC-0100_BaseLayer.md)
+* [RFC-0100: Base Layer](RFC-0100_BaseLayer.md)
 
 ## Description
 
-### Broad requirements
+### Broad Requirements
 
 Tari Base Nodes form a peer-to-peer network for a proof-of-work based blockchain running the [Mimblewimble]
-protocol. The proof of work is performed via merge mining with Monero. Arguments for this design are
+protocol. The proof-of-work is performed via merge mining with Monero. Arguments for this design ar
 presented [in the overview](RFC-0001_overview.md#proof-of-work).
 
 Tari Base Nodes MUST carry out the following tasks:
 
-* Validate all [Tari coin] [transaction]s.
-* Propagate valid transactions to peer nodes.
-* Validate all new [block]s received.
-* Propagate validated new blocks to peer nodes.
-* Connect to peer nodes to catch up (sync) its blockchain state.
-* Provide historical block information to peers that are syncing.
+* validate all [Tari coin] [transaction]s;
```

Comment:
`[Tari coin] [transaction]s` -> `[Tari coin] transactions`

The two hyperlinks next to each other do not render correctly in mdbook